components/src/Select/Select.js

```diff
@@ -152,7 +153,7 @@ const Select = ({
         highlightedIndex
       }) => (
         <div style={{ position: "relative" }}>
-          <SelectBox {...getToggleButtonProps({ disabled, error, isOpen })}>
+          <SelectBox {...getToggleButtonProps({ disabled, error: !!(errorMessage || errorList), isOpen })}>
```

Comment:
Looks like `!!(errorMessage || errorList)` appears in a *lot* of places.
Definitely an opportunity for cleanup.

Reply:
Did some cleanup here